basis/ui/backend/gtk/gtk.factor

```diff
@@ -420,9 +425,11 @@ M: window-handle flush-gl-context ( handle -- )
 
 M:: gtk-ui-backend (open-window) ( world -- )
     GTK_WINDOW_TOPLEVEL gtk_window_new :> win
+    gtk_drawing_area_new :> drawable
+    win drawable gtk_container_add
```

Comment:
Yes kind of. We only need gtk to give us an opengl context. So before we were calling gtk_widget_set_gl_capability and gtk_widget_get_gl_context on the window. The window had no children.

Now the window only has 1 child (the child has no children). We call the 2 gtk functions on the child.

So the only difference is that those 2 contexts are treated differently inside the gtk library (see https://git.gnome.org/browse/gtkglext/tree/gdk/gdkglwindow.c vs https://git.gnome.org/browse/gtkglext/tree/gdk/gdkglpixmap.c I think)